SQL/0000-00-00-schema.sql

```diff
@@ -1552,20 +1551,20 @@ CREATE TABLE `parameter_type` (
 
 
 INSERT INTO `parameter_type` VALUES
```

Comment:
```suggestion
INSERT INTO `parameter_type` (ParameterTypeID, Name, Type, Description, RangeMin, RangeMax, SourceField, SourceFrom, SourceCondition, Queryable, IsFile) VALUES
```
with some lint for line lenght

Reply:
@PapillonMcGill your parameter count does not match the number of parameters in the INSERT statements.

an addition like that should be properly tested before submission or travis will most probably fail

Reply:
I will make the appropriate changes and add it to my PR

Reply:
@ridz1208 sorry, Done the suggestion quickly while on a conf. call.